spec/bundler/shared_helpers_spec.rb

```diff
@@ -121,6 +121,14 @@
       end
     end
 
+    context "ENV['BUNDLE_GEMFILE'] is set without an absolute path" do
+      before { ENV["BUNDLE_GEMFILE"] = "Gemfile" }
+
+      it "returns path of the bundle Gemfile" do
+        expect(subject.in_bundle?).to eq("#{bundled_app}/Gemfile")
```

Comment:
please test that `subject.find_gemfile` returns this, as only the truthiness of `in_bundle?` is guaranteed.